patches/third_party-blink-renderer-build-scripts-core-css-properties-templates-style_builder_functions.tmpl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl b/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
+index 5ff29d8618a2261fcf92c9a04471d21ecef5b8af..5b7af3f60227f66591c3ee8e959a08c1fdaa2e6e 100644
+--- a/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
++++ b/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
+@@ -18,7 +18,7 @@ void {{prefix ~ '::' if prefix}}ApplyValue(StyleResolverState& state, const CSSV
+     {% if property.style_builder_inline == header and property.style_builder_generate_initial %}
+ {{declare_initial(property.name.to_upper_camel_case() if not header)|indent(spaces, true)}} {
+ {{caller(property)|indent(spaces, true)-}}
+-{{'}'|indent(spaces, true)}}
```

Comment:
actually this only solves 1 generated function `ApplyInherit`, the other two `ApplyInitial` and `ApplyValue` are still like this `// Intentionally empty.}`
I think we need this
```
--- a/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
+++ b/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
@@ -18,7 +18,7 @@ void {{prefix ~ '::' if prefix}}ApplyValue(StyleResolverState& state, const CSSV
     {% if property.style_builder_inline == header and property.style_builder_generate_initial %}
 {{declare_initial(property.name.to_upper_camel_case() if not header)|indent(spaces, true)}} {
 {{caller(property)|indent(spaces, true)-}}
-{{'}'|indent(spaces, true)}}
+{{'\n}'|indent(spaces, true)}}
     {% else %}
 {{declare_initial()|indent(spaces, true)-}};
     {% endif %}
@@ -31,7 +31,7 @@ void {{prefix ~ '::' if prefix}}ApplyValue(StyleResolverState& state, const CSSV
     {% if property.style_builder_inline == header and property.style_builder_generate_inherit %}
 {{declare_inherit(property.name.to_upper_camel_case() if not header)|indent(spaces, true)}} {
 {{caller(property)|indent(spaces, true)-}}
-{{'}'|indent(spaces, true)}}
+{{'\n}'|indent(spaces, true)}}
     {% else %}
 {{declare_inherit()|indent(spaces, true)-}};
     {% endif %}
@@ -44,7 +44,7 @@ void {{prefix ~ '::' if prefix}}ApplyValue(StyleResolverState& state, const CSSV
     {% if property.style_builder_inline == header and property.style_builder_generate_value %}
 {{declare_value(property.name.to_upper_camel_case() if not header)|indent(spaces, true)}} {
 {{caller(property)|indent(spaces, true)-}}
-{{'}'|indent(spaces, true)}}
+{{'\n}'|indent(spaces, true)}}
     {% else %}
 {{declare_value()|indent(spaces, true)-}};
     {% endif %}
```